mantidproject/mantid
# Label groups with run title in Reflectometry (Polref) GUI

When runs are transferred to the processing table, the group should include a description. There are two "strategies" to transfer runs, `Measurement` and `Description`, which can be selected using the combo box next to the `Search` button. Changes in this PR apply to both cases.

**To test:**

- Check code changes. I had to modify `LoadNexusLogs` so that `run_title` is added to the workspace.
- Functional testing: I think the best way to test the changes is by comparing results to the master branch:
  - Measurement strategy:
    - Open MantidPlot -> Interfaces -> Reflectometry -> ISIS Reflectometry (Polref)
    - Select `POLREF` in the instrument combo box. Select `Measurement` strategy.
    - Enter `1520249` and use your user ID and password to log in to the catalog. The table on the left will be populated with some runs. Select all of them and click on `Transfer`, runs will appear on the table on the right. On the master branch there will be an integer number associated with each group. On this branch the integer number should be followed by a description.
    - Expand each group and check that they contain the same run numbers and angles as on the master branch.
  - Legacy strategy:
    - Repeat the same steps but selecting `Description` strategy (on the master branch you may need to close and re-open the interface due to a known bug I have fixed on this branch). This time, integer numbers should not appear, only group information.
    - Expand groups and check that they contain the same run numbers and angles as the master branch (note that because group title does not include the integer number, groups will have been transferred in a different order)

Fixes #17817 

**Release notes** were updated.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
